cluster/saltbase/salt/kube-addons/init.sls

```diff
@@ -0,0 +1,79 @@
+{% if pillar.get('enable_cluster_monitoring', '').lower() == 'true' %}
```

Comment:
I bet I know what's wrong. I bet the commit that changed the init.sls YAML to all be quoted actually broke this, because this looks like
`enable_cluster_monitoring: 'true'`
now, which is probably coming in as a Python string, no? (I need to get up a cluster to test this, jussec)